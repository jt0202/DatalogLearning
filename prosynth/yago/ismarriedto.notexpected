napoleon_iii	robert_fisk
frederick_william_iii_of_prussia	ansgar_von_lahnstein
maud_fitzjohn	bijou_heron
princess_olga_paley	james_rubin
amarindra	lt_Duke_of_FlorenceMargaret_of_Parmagt
