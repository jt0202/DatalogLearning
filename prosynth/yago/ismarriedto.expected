liang_sicheng	lin_huiyin
nash_brennan	jessica_buchanan
jeffrey_oneill	olivia_spencer
akhenaten	ankhesenamun
travis_montgomery	erica_kane
hayley_vaughan	mateo_santos
blair_cramer	todd_manning
harley_cooper	phillip_spaulding
jessica_buchanan	nash_brennan
erica_kane	dimitri_marick
tad_martin	brooke_english
beth_rainesspaulding	phillip_spaulding
jennifer_rappaport	joey_buchanan
joey_buchanan	jennifer_rappaport
krystal_carey	david_hayward
ruth_martin_all_my_children	joe_martin_all_my_children
olivia_spencer	phillip_spaulding
natalia_rivera_aitoro	gus_aitoro
brooke_english	tad_martin
gus_aitoro	natalia_rivera_aitoro
marissa_tasker	jr_chandler
gloria_marsh	adam_chandler
phillip_spaulding	harley_cooper
lin_huiyin	liang_sicheng
jeff_martin_all_my_children	erica_kane
olivia_winters	nathan_hastings
iset_daughter_of_amenhotep_iii	amenhotep_iii
